mozdef_util/mozdef_util/elasticsearch_client.py

```diff
@@ -38,6 +38,12 @@ def __init__(self, servers, bulk_amount=100, bulk_refresh_time=30):
         self.es_connection.ping()
         self.bulk_queue = BulkQueue(self, threshold=bulk_amount, flush_time=bulk_refresh_time)
 
+    def close_index(self, index_name)
+        self.es_connection.indices.close(index=index_name)
+
+    def open_index(self, index_name)
```

Comment:
Syntax error, missing a trailing semi colon.

Reply:
fixed in last commit.